src/components/Header/Header.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  hasActiveTodos: boolean,
+  title: string,
+  onTitleChange: (newTitle: string) => void,
+};
+
+export const Header: React.FC<Props> = ({
+  hasActiveTodos,
+  title,
+  onTitleChange,
+}) => (
+  <header className="todoapp__header">
+    <button
+      type="button"
+      className={classNames(
+        'todoapp__toggle-all',
+        { active: !hasActiveTodos },
+      )}
+    />
```

Comment:
button is visible only if length of todos > 0, lets fix it https://prnt.sc/Hx0dUcJ5kLtQ

Reply:
Thanks, get it, will fix it in next part of this task!)